turbinia/e2e/setup-gce.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# This script will
+# - setup a GCE Ubuntu 18.04 LTS base image for Turbinia e2e testing.
+# - execute Terraform to setup the Turbinia environment.
+# Make sure the service account your GCE instance is running under has full API scope
+# access and is project owner for Terraform to function correctly.
+echo "Initiate gcloud configuration"
+gcloud info
+
+apt-get update
+apt-get -y install python-pip python-virtualenv unzip
```

Comment:
I guess virtualenv/pip is a requirement for the `deploy.sh` script?

Reply:
Correct, it installs Turbinia in a virtualenv.